extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -505,27 +505,23 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
   }
 
   test("[KYUUBI #3343] pass temporary view creation") {
```

Comment:
btw, we can move this test case to superclass ?

Reply:
If just checking view privileges on select values, consider move to PrivilegesBuilderSuite？
Put ut here is to check views with hive input tables.